docs-markdown/package.json

```diff
@@ -311,6 +311,7 @@
     "test-core": "node ./node_modules/vscode/bin/test"
   },
   "dependencies": {
+    "@types/jest": "^24.0.11",
```

Comment:
This should be a dev dependency with the other types

Reply:
Good catch.

Reply:
@bharney-msft - updated!